app/assets/javascripts/orders.coffee

```diff
@@ -17,7 +17,7 @@ showOrderDialog = (orderId) ->
     error: (jqXHR, textStatus, errorThrown) ->
       alert "Error occurred"
 
-window.orderRowClicked = (event, row, element) ->
+orderRowClicked = (event, row, element) ->
```

Comment:
I went ahead and did this in #184 because I noticed the need for this in a comment on another PR.